src/index.ts

```diff
@@ -706,8 +738,8 @@ server.setRequestHandler(ReadResourceRequestSchema, async (request) => {
     const uri = request.params.uri;
     const parts = uri.replace("k8s://", "").split("/");
 
-    if (parts[0] === "namespaces" && parts.length === 1) {
-      const { body } = await k8sManager.getCoreApi().listNamespace();
+    if ((parts[0] === "namespaces" || parts[0] === "nodes") && parts.length === 1) {
+      const { body } = await k8sManager.getCoreApi()[parts[0] === "nodes" ? "listNode" : "listNamespace"]();
```

Comment:
Tiny nit to make this cleaner (i'll clean it up in future commit, would need to make sure it works w/ typescript).

```suggestion
    const isNamespaces = parts[0] === "namespaces";
    const isNodes = parts[0] === "nodes";
    if ((isNamespaces || isNodes) && parts.length === 1) {
      const fn = isNodes ? "listNode" : "listNamespace";
      const { body } = await k8sManager.getCoreApi()[fn]();
```

Reply:
LGTM otherwise, will merge & make new release tomorrow since haven't setup ci/cd.